frontend/src/Pages/OneRecipe.jsx

```diff
@@ -70,12 +70,12 @@ const OneRecipe = () => {
   // Function to handle posting a comment
   const handlePostComment = async () => {
     // getting username from token named "username" created during login
-    const username = JSON.parse(localStorage.getItem("username"));
+    const token = JSON.parse(localStorage.getItem("tastytoken"));
     if (commentInput.trim()) {
       try {
         await axios.post(`${backendURL}/api/recipe/addcomment`, {
           recipeId: recipe._id,
-          username: username,  // Assuming the user is already logged in
+          token: token,  // Assuming the user is already logged in
           content: commentInput,
         },
```

Comment:
@haseebzaki-07 Do not assume, check if the user is logged in, If the user is logged in then only let them submit the feedback form.

Reply:
@AlfiyaSiddique the token is retrieved from the localStorage of the application, that is set only after the user logs in, so retreiveing the token means the user is logged in and is good to comment.

Reply:
oaky great!